tests/test_widgets.py

```diff
@@ -3530,6 +3534,62 @@ def test_textbox_autoscroll(self):
             "|                                      |\n" +
             "+--------------------------------------+\n")
 
+    def test_layout_gutters(self):
+        """
+        Validate that the gutters parameter works for Layouts
+        """
+        # Now set up the Frame ready for testing
+        screen = MagicMock(spec=Screen, colours=8, unicode_aware=False)
+        scene = Scene([], duration=-1)
+        canvas = Canvas(screen, 10, 40, 0, 0)
+        form = Frame(canvas, canvas.height, canvas.width)
+        layout = Layout([18, 4, 18], gutter=3)
+        form.add_layout(layout)
+
+        #       0
+        text = "ABCDEFGHIJKLMNOP"
+
+        box1 = TextBox(1)
+        box1.value = [text]
+        layout.add_widget(box1, 0)
+
+        box2 = TextBox(1)
+        box2.value = [text]
+        layout.add_widget(box2, 1)
+
+        box3 = TextBox(1)
+        box3.value = [text]
+        layout.add_widget(box3, 2)
+
+        form.fix()
+        scene.add_effect(form)
+        scene.reset()
+
+        # Check that the frame is rendered correctly.
+        for effect in scene.effects:
+            effect.update(0)
+
+        print("**********")
```

Comment:
Any reason for printing this out?  That's what the assert is for...  Right?

Reply:
Whole chunk is debug, it has been removed.